src/lib/sql/sql_translator.cpp

```diff
@@ -1214,6 +1217,16 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_import(const hsql::Im
                           import_type_to_file_type(import_statement.type));
 }
 
+// NOLINTNEXTLINE - while this particular method could be made static, others cannot.
+std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_export(const hsql::ExportStatement& export_statement) {
+  // Get validated table as input
+  auto sql_identifier_resolver = std::make_shared<SQLIdentifierResolver>();
+  auto lqp = _translate_stored_table(export_statement.tableName, sql_identifier_resolver);
+
+  return ExportNode::make(export_statement.tableName, export_statement.filePath,
+                          import_type_to_file_type(export_statement.type), lqp);
```

Comment:
import_type sounds strange in the context of an export.

Reply:
Actually, the parser uses an ImportType to indicate the file type both for ImportStatement and ExportStatement due to we wanted to be downwards compatible there

Reply:
So it is an ImportExportType? But it's probably not worth changing it.

Reply:
We use it for both, exactly, but its name in the parser stayed ImportType so others would not need to adapt their code if they use an ImportStatement anywhere.